admin/class-admin.php

```diff
@@ -312,8 +312,21 @@ protected function set_upsell_notice() {
 
 	/**
 	 * Initializes Whip to show a notice for outdated PHP versions.
+	 *
+	 * @todo Deprecate this method when WordPress 5.1 is our currently minimal supported version.
+	 *
+	 * @return void
 	 */
 	protected function check_php_version() {
+		/*
+		 * The Whip message shouldn't be shown from WordPress 4.9.5 and higher because
+		 * that version introduces Serve Happy which is almost similar to Whip.
+		 */
+		$minimal_required_wordpress_version = '4.9.5';
+		if ( version_compare( $GLOBALS['wp_version'], $minimal_required_wordpress_version, '>=' ) ) {
```

Comment:
You could make Codeclimate happy by renaming $minimal_required_wordpress_version to a name with max 20 chars. Also: Is it required to keep the WordPress notation (with the capital W & P) in function names?